_examples/bundler/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:stretch
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    cmake \
+    libcurl4-openssl-dev \
+    libicu-dev \
+    libmagic-dev \
+    libmariadbclient-dev \
+    libsqlite3-dev \
+    libsystemd-dev \
+    pkg-config \
+    ruby ruby-dev ruby-bundler \
+    zlib1g-dev
```

Comment:
Are we sure all these dependencies are needed? icu, libmagic, libmariadbclient etc. seem irrelevant.

Reply:
Some of them are redundant indeed.